### DISCUSSION TITLE
Do I have bad naming of tracks or tags?

### DISCUSSION THREAD

**Discussion by VeksCZ** [tranxuanthang/lrcget#108] [Q&A] [answered]
![obrazek](https://github.com/tranxuanthang/lrcget/assets/135608180/fec46963-ce86-47da-aafb-6be6cc04b90d)
This tool is great but cannot find many of tracks lyrics. I can't find any setting or advice for track filename or tag formatting.

**Comment by VeksCZ** [comment]
![obrazek](https://github.com/tranxuanthang/lrcget/assets/135608180/9403f196-f640-4aef-b67f-1ac71a67c344)
Strange.. I had it running for 10000+ songs and after a while I had some error. So I checked skip already downloaded lyrics and now I have much better results with finding lyrics.

**Comment by tranxuanthang** [comment, marked as answer]
Each time when a requested track is not found in database, LRCLIB will add its metadata to a queue, and later try to retrieve the lyrics from 3rd party services. That's the reason why you will usually find a lot more results on the second download, and it is expected behavior!

**Comment by Papiaceituna28** [comment]
¿Hay alguna otra base de datos que recomienden?
